### PR TITLE
6.1.0

### PR DESCRIPTION
**Pull request by rekmarks**
- `6.1.0`
- Update changelog
  - #72, #73, #76, #78

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.0] - 2020-07-21
+
+### Changed
+
+- Only emit `data` event for notifications present in `^4.0.0` ([#73](https://github.com/MetaMask/inpage-provider/pull/73))
+  - Some consumers make assumptions about the shape of the object emitted with the event that do not hold all notifications in `^6.0.0`.
```
Comment (Gudahtt) [inline CHANGELOG.md]: Typo I think:

```suggestion
  - Some consumers make assumptions about the shape of the object emitted with the event that do not hold for all notifications in `^6.0.0`.
```

Comment (rekmarks) [reply CHANGELOG.md]: a2e7626

**Review by Gudahtt — COMMENTED** [review]
https://github.com/MetaMask/inpage-provider/pull/72 probably deserves mention too

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - Some consumers make assumptions about the shape of the object emitted with the event that do not hold for all notifications in `^6.0.0`.
+- Select icon using `rel~="icon"` when retrieving site metadata
+([#76](https://github.com/MetaMask/inpage-provider/pull/76))
+  - This is instead of defaulting to `rel="shortcut icon"`.
+
+### Fixed
+
+- Enable retrieval of site icons _not_ hosted on the same origin
+([#78](https://github.com/MetaMask/inpage-provider/pull/78))
+  - For example, icons hosted on `assets.foo.com` that are used on `foo.com` will now be retrieved successfully.
```
Comment (Gudahtt) [inline CHANGELOG.md:28]: e.g.
```suggestion
  - For example, icons hosted on `assets.foo.com` that are used on `foo.com` will now be retrieved successfully.
- Emit events after all state has been updated ([#72](https://github.com/MetaMask/inpage-provider/pull/72))
```

Comment (rekmarks) [reply CHANGELOG.md:28]: aed0471 😅 🏁

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**